pom.xml

```diff
@@ -520,6 +520,11 @@
       <groupId>org.apache.xmlbeans</groupId>
       <artifactId>xmlbeans</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Janino is needed: https://logback.qos.ch/manual/configuration.html#conditional
- [ ] Need to check availability